src/locales/en/translations.json

```diff
@@ -1,5 +1,8 @@
 {
-    "Please provide a valid date": "",
+    "Choose one or more dates...": "",
```

Comment:
Will have to ask again, how this is populated here?  I mean is it appearing when you run `yarn start`?

Reply:
yes, yarn start and build. The file was modified now because by moving files around the translation order changed (I think)